internal/tiltfile/files.go

```diff
@@ -125,8 +125,8 @@ func (s *tiltfileState) execLocalCmd(t *starlark.Thread, cmd model.Cmd, options
 }
 
 func (s *tiltfileState) kustomize(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	var path starlark.Value
-	err := s.unpackArgs(fn.Name(), args, kwargs, "paths", &path)
+	var path, kustomizeBin starlark.Value
```

Comment:
can you make these both a `NewLocalPathUnpacker()`?

Since this code was written, starlark supports custom unpackers, so the deserialization is built into the type object.